docker/master-java/Dockerfile

```diff
@@ -18,19 +18,16 @@ RUN apt-get update && \
       libmpfr-dev \
       libsqlite3-dev \
       make \
-      openjdk-8-jdk-headless \
+      openjdk-11-jdk-headless \
```

Comment:
Did you test if infer works with java11? Eg `infer -- javac examples/Hello.java`.

Reply:
Yes, it works.